tanner/tests/test_rfi_emulation.py

```diff
@@ -1,8 +1,17 @@
 import asyncio
 import unittest
-
+from unittest import mock
 from tanner.emulators import rfi
 
+# Custom function to mock ascyncio coroutines
+def AsyncMock(*args, **kwargs):
```

Comment:
lowercase with words separated by underscores 
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names